CMSgov/design-system
# [WNMGDS-958] bash script for testing DS changes locally in other projects

## Summary
Based off a script that Kimberly used frequently to test design system changes in other applications, I created a bash script where it installs & build local dependencies for child system or application testing.

### Added
Script for local testing

## How to test
_Please note that this script assumes your core ds project, child systems, and applications all share a parent directory._

To test with script default, run `./test-locally.sh` in the root directory of the core DS. This should start a build that will
1. install and build core
2. install and build the mgov design system
3. install and build coverage tools project

This script also has the option to overwrite the app or child system paths when running the script. For example, if I wanted to test hcgov child system, I could run 
```
./test-locally.sh -c hcgov-design-system -a windowshop
```
where `-c` tells the directory name for the child system & `-a` tells the directory name for the application.

## Outstanding questions
1. Is this a helpful script?
2. What would make it more helpful?
3. Does the setup make sense?
